try:
    while True:
        pass
except Exception as ex:
    print(ex)
    print("Exception Occured")